packages/site/package.json

```diff
@@ -4,77 +4,66 @@
-    "@pluralsight/ps-design-system-starrating": "^3.0.0",
-    "@pluralsight/ps-design-system-switch": "^4.0.0",
-    "@pluralsight/ps-design-system-tab": "^12.0.0",
-    "@pluralsight/ps-design-system-table": "^5.0.0",
-    "@pluralsight/ps-design-system-tag": "^4.0.0",
-    "@pluralsight/ps-design-system-text": "^13.0.0",
-    "@pluralsight/ps-design-system-textarea": "^3.0.0",
-    "@pluralsight/ps-design-system-textinput": "^3.0.0",
-    "@pluralsight/ps-design-system-theme": "^3.0.3",
-    "@pluralsight/ps-design-system-tooltip": "^4.0.6",
-    "@pluralsight/ps-design-system-util": "^2.0.5",
-    "@pluralsight/ps-design-system-viewtoggle": "^4.0.0",
-    "babel-core": "^6.25.0",
-    "babel-plugin-syntax-export-extensions": "^6.13.0",
-    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "@pluralsight/ps-design-system-actionmenu": "*",
+    "@pluralsight/ps-design-system-avatar": "*",
+    "@pluralsight/ps-design-system-badge": "*",
+    "@pluralsight/ps-design-system-banner": "*",
+    "@pluralsight/ps-design-system-breadcrumb": "*",
+    "@pluralsight/ps-design-system-button": "*",
+    "@pluralsight/ps-design-system-card": "*",
+    "@pluralsight/ps-design-system-carousel": "*",
+    "@pluralsight/ps-design-system-checkbox": "*",
+    "@pluralsight/ps-design-system-circularprogress": "*",
+    "@pluralsight/ps-design-system-core": "*",
+    "@pluralsight/ps-design-system-datawell": "*",
+    "@pluralsight/ps-design-system-datepicker": "*",
+    "@pluralsight/ps-design-system-dialog": "*",
+    "@pluralsight/ps-design-system-drawer": "*",
+    "@pluralsight/ps-design-system-dropdown": "*",
+    "@pluralsight/ps-design-system-emptystate": "*",
+    "@pluralsight/ps-design-system-errors": "*",
+    "@pluralsight/ps-design-system-form": "*",
+    "@pluralsight/ps-design-system-icon": "*",
+    "@pluralsight/ps-design-system-layout": "*",
+    "@pluralsight/ps-design-system-linearprogress": "*",
+    "@pluralsight/ps-design-system-link": "*",
+    "@pluralsight/ps-design-system-normalize": "*",
+    "@pluralsight/ps-design-system-note": "*",
+    "@pluralsight/ps-design-system-position": "*",
+    "@pluralsight/ps-design-system-radio": "*",
+    "@pluralsight/ps-design-system-row": "*",
+    "@pluralsight/ps-design-system-searchinput": "*",
+    "@pluralsight/ps-design-system-starrating": "*",
+    "@pluralsight/ps-design-system-switch": "*",
+    "@pluralsight/ps-design-system-tab": "*",
+    "@pluralsight/ps-design-system-table": "*",
+    "@pluralsight/ps-design-system-tag": "*",
+    "@pluralsight/ps-design-system-text": "*",
+    "@pluralsight/ps-design-system-textarea": "*",
+    "@pluralsight/ps-design-system-textinput": "*",
+    "@pluralsight/ps-design-system-theme": "*",
+    "@pluralsight/ps-design-system-tooltip": "*",
+    "@pluralsight/ps-design-system-util": "*",
+    "@pluralsight/ps-design-system-viewtoggle": "*",
+    "prop-types": "^15.7.2"
+  },
+  "dependencies": {
     "babel-preset-env": "^1.6.0",
```

Comment:
do we still need these babel presets? we have removed our `.babelrc` file from the package root so i don't think that we'd be doing any custom babel config.

there is also a `next/babel` preset that is suggested in the docs but not sure what it does.

Reply:
babel-standalone uses plugins these inside the eval for code inside the live code component.  It's not for next per se.  It's used in the browser.